### DISCUSSION TITLE
ETA for UE5 Plugin?

### DISCUSSION THREAD

**Discussion by sommerlost** [Sixze/ALS-Refactored#83] [General]
Hey @Sixze great work on the plugin. I was curious if you have any ETA or roadmap for the UE5 release? Thanks!

**Comment by Sixze** [comment]
The plugin is ready for Unreal Engine 5, but before releasing a new version, I want to fix a couple of minor bugs.

**Comment by Error0xFF** [comment]
hi there! im new to unreal engine and i explored the original ALS before discovering your version. i want to thank you for the effort you put into developing this into a clean and optimized version. i picked it as a starting point for my learning project and everything is going very well. also is it possible to expose some of the c++ variables to blueprints? like for example the "movement settings" variable? im loading that from a data table custom tailor for each character class. worderful job again thank you!

**Reply by Sixze** [reply]
Use the `AlsCharacterMovementComponent::SetMovementSettings()` function.

**Reply by Error0xFF** [reply]
Thank you! Do you have a discord server?
